suites/suite_cufflinks_2_2_1/.shed.yml

```diff
@@ -0,0 +1,2 @@
+owner: devteam
```

Comment:
@martenson this should have more informations, right? Would it be worth to have an own category for suites?